update_repos

```diff
@@ -88,10 +91,11 @@ def read_api_uri(uri, config, page=1):
     }
 
     if config.debug:
-        print "Trying URI {} with headers {}".format(uri, headers)
+        print("Trying URI {} with headers {}".format(uri, headers))
 
     req = Request(uri, headers=headers)
-    return urlopen(req).read()
+    encoding = locale.getpreferredencoding()
+    return urlopen(req).read().decode(encoding)
```

Comment:
I think if you don't pass an argument to `decode` it does this automatically, but it didn't hurt. The other potential fix would be to get the encoding once globally during initialization since it's not going to change during the lifetime of the process. This is fine, though.

Reply:
Changed it to be a global variable.